apps/namadillo/src/App/Transactions/TransactionCard.tsx

```diff
@@ -125,7 +125,9 @@ export const TransactionCard = ({
     : getTransactionInfo(transaction);
   const baseAmount =
     asset && txnInfo?.amount ?
-      toDisplayAmount(asset, txnInfo.amount)
+      isBondingTransaction ? toDisplayAmount(asset, txnInfo.amount)
+      : isNamadaAsset(asset) ? txnInfo.amount
+      : toDisplayAmount(asset, txnInfo.amount)
```

Comment:
this is a ternary hell to read, maybe we could create a helper function with "return earlier" to handle this